packages/apm/src/span.ts

```diff
@@ -307,6 +306,11 @@ export class Span implements SpanInterface, SpanContext {
       }).timestamp;
     }
 
+    // We will not send any child spans
+    if (this.isChildSpan()) {
+      return undefined;
```

Comment:
(style question) Why do we write `return undefined;` everywhere instead of `return;`?

Reply:
TypeScript